lib/bytes_by_prefix.c

```diff
@@ -62,12 +62,16 @@
 #define B_MULT	BSIZE		/* block size */
 #elif sgi
 #define B_MULT	BBSIZE		/* block size */
-#elif defined(__linux__) || defined(__sun) || defined(__hpux)
+#elif (defined(__linux__) || defined(__sun) || defined(__hpux)) && defined(DEV_BSIZE)
 #define B_MULT	DEV_BSIZE	/* block size */
```

Comment:
Sorry for not being clear enough. LTP does not support anything but Linux at the moment (minus the open posix testsuite that is written accordingly to the POSIX standard). So we can get rid of everything but the value used for Linux. So something as:

```c
#ifdef DEV_BSIZE
# define B_MULT DEV_BSIZE
#else
# warning Unable to find a good value for B_MULT, defaulting to 512
# define B_MULT 512
#endif
```